static_objects:
 - calib_board_1:
    pose: 
      center:
        x: 0.00
        y: 0.00
        z: 0.00
      rotvec:
        x: 0.000000
        y: 0.0
        z: 0.0
      position_noise:
        x: 0.002000
        y: 0.002000
        z: 0.001000
      rotation_noise:
        x: 0.000100
        y: 0.000100
        z: 0.000100
    tags:  
    - id: 0
      size: 0.04
      center:
        x: 0.02
        y: 0.02
        z: 0.00
      rotvec:
        x: 0.000000
        y: 0.0
        z: 0.0
      position_noise:
        x: 0.002000
        y: 0.002000
        z: 0.001000
      rotation_noise:
        x: 0.000100
        y: 0.000100
        z: 0.000100
